test_main.py

```diff
@@ -0,0 +1,2 @@
+def test_sum():
+    assert True
```

Comment:
нет переноса строки в конце файла
```suggestion
    assert True
    
```